pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -24,8 +24,7 @@
 
   hoc2018_tutorial_mchoc_description: 'Minecraft is back for the Hour of Code with a brand new activity! Journey through Minecraft with code.'
 
-  hoc2018_minecraft_aquatic_title1: "Minecraft"
-  hoc2018_minecraft_aquatic_title1: "Voyage Aquatic"
+  hoc2018_minecraft_aquatic_title1: "Minecraft Voyage Aquatic"
```

Comment:
How does this lead to the "after" screenshot that is provided?

Reply:
Before my change these two strings had the same key, but it looks like it probably used `Voyage Aquatic` since that came later.

Changing this to one key so the key isn't duped. And fixed it to the whole title. So the before has `Voyage Aquatic` and the after has `Minecraft Voyage Aquatic`.

Reply:
Oh right. I was thinking the intent was to shorten rather than lengthen the string.